ethereum/listener.go

```diff
@@ -55,43 +56,49 @@ func (e *Ethereum) StartListener(
 		Ready: make(chan struct{}),
 	}
 
-	// start main stream (does not account for lookback period or specific start block)
-	stream, sub, history := e.startMainStream(ctx, logger, messageSent, messageTransmitterAddress)
+	// FlushOnlyMode should only run the flush mechanism, otherwise start the main listener,
+	// otherwise consume history and incoming msgs
```

Comment:
this comment reads kind of funny

Reply:
Oh wow, I'm not sure what I was even thinking there 🤦‍♂️